.gitlab-ci.yml

```diff
@@ -153,14 +153,17 @@ test-on-netgear-rax40:
   variables:
     # TESTS_TO_RUN needs to be set by the user (or the pipeline schedule)
     GIT_CLONE_PATH: "/builds/prpl-foundation/prplMesh/"
+    DEVICE_UNDER_TEST: prplmesh
   script:
-      - /easymesh_cert/run_test_file.sh -o logs $TESTS_TO_RUN
+      - tools/deploy_ipk.sh prplmesh-rax40 tools/docker/builder/openwrt/*.ipk
```

Comment:
Shouldnt this be
```suggestion
      - tools/deploy_ipk.sh $DEVICE_UNDER_TEST tools/docker/builder/openwrt/*.ipk
```

Reply:
No, because if the DEVICE_UNDER_TEST is 'prplmesh', we don't need to deploy anything.
I agree we could use a conditional and deploy to DEVICE_UNDER_TEST in all other cases though.

Reply:
Done.